podcasts/End of Year/Animations/PlayPauseAnimatableModifier.swift

```diff
@@ -12,8 +12,15 @@ class PlayPauseAnimationViewModel: ObservableObject {
         self.animationType = animation
     }
 
-    func animate(_ value: Binding<Double>, to: Double) -> PlayPauseAnimatableModifier {
-        return PlayPauseAnimatableModifier(value: value, to: to, duration: duration, viewModel: self, animation: animationType)
+
+    /// Adds a modifier that will animates the given View
+    /// - Parameters:
+    ///   - value: the value to bind to
+    ///   - to: the final value of the property
+    ///   - after: how long the animation should wait before starting
+    /// - Returns: PlayPauseAnimatableModifier
+    func animate(_ value: Binding<Double>, to: Double, after: Double = 0) -> PlayPauseAnimatableModifier {
```

Comment:
Right now this doesn't handle the duration of the animation that starts `after` some time. It will have the same duration as when `PlayPauseAnimationViewModel` is created.

I did that to avoid adding more code here.